code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -51,7 +51,7 @@
 /obj/item/weapon/cartridge/Initialize()
 	. = ..()
 	var/obj/item/device/pda/pda = loc
-	if(istype(pda, /obj/item/device/pda))
+	if(istype(pda))

```

Comment:
the type of the variable ```pda``` is ```/obj/item/device/pda``` (see: ```var/obj/item/device/pda/pda = loc```), if ```istype()``` is not provided a 2nd argument, it instead uses the type of the variable.

This kind of ```istype()``` is basically ```is this variable what I think it is?``` as opposed to ```is this variable an instance of X?```